sw/device/tests/alert_handler_ping_timeout_test.c

```diff
@@ -138,20 +137,39 @@ void ottf_external_isr(void) {
   isr_testutils_alert_handler_isr(plic_ctx, alert_handler_ctx,
                                   &peripheral_serviced, &irq_serviced);
   CHECK(peripheral_serviced == kTopEarlgreyPlicPeripheralAlertHandler,
-        "Interurpt from unexpected peripheral: %d", peripheral_serviced);
+        "Interrupt from unexpected peripheral: %d", peripheral_serviced);
+
+  // Only interrupts from class B alerts are expected for this test. Report the
+  // unexpected alert(s).
```

Comment:
Hey Alex, this might not always be the case.
Because the ping timeout is set to a small value, depending on the handshake timing, we might get spurious alert.
Please see issue: https://github.com/lowRISC/opentitan/issues/2321

Reply:
Thanks for the info!